packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -41,6 +42,29 @@ export function buildRootObject(vatPowers) {
         repeaters.set(index, vatRepeater);
         return vatRepeater;
       },
+      createNotifier(delaySecs, interval) {
```

Comment:
Should this be called `createNotifier` or `makeNotifier`? I thought we generally wanted to use `make`.

Reply:
Please rename both, but leave the old one too, as deprecated.